jmclient/jmclient/configure.py

```diff
@@ -141,56 +147,20 @@ def jm_single():
 #socks5_host = localhost
 #socks5_port = 9050
 
-## SERVER 2/4) hackint IRC (Tor, IP)
-################################################################################
-[MESSAGING:server2]
-channel = joinmarket-pit
-
-# for traditional IP (default):
-host = irc.hackint.org
-port = 6697
-usessl = true
-socks5 = false
-
-# for Tor (recommended as clearnet alternative):
-#host = ncwkrwxpq2ikcngxq3dy2xctuheniggtqeibvgofixpzvrwpa77tozqd.onion
-#port = 6667
-#usessl = false
-#socks5 = true
-#socks5_host = localhost
-#socks5_port = 9050
-
-## SERVER 3/4) Anarplex IRC (Tor, IP)
-################################################################################
-[MESSAGING:server3]
-channel = joinmarket-pit
-
-# for traditional IP (default):
-host = agora.anarplex.net
-port = 14716
-usessl = true
-socks5 = false
-
-# for Tor (recommended as clearnet alternative):
-#host = vxecvd6lc4giwtasjhgbrr3eop6pzq6i5rveracktioneunalgqlwfad.onion
-#port = 6667
-#usessl = false
-#socks5 = true
-#socks5_host = localhost
-#socks5_port = 9050
-
-## SERVER 4/4) ILITA IRC (Tor — disabled by default)
-################################################################################
-#[MESSAGING:server4]
-#channel = joinmarket-pit
-#port = 6667
-#usessl = false
-#socks5 = true
-#socks5_host = localhost
-
-# for Tor (recommended):
-#host = ilitafrzzgxymv6umx2ux7kbz3imyeko6cnqkvy4nisjjj4qpqkrptid.onion
-#socks5_port = 9050
```

Comment:
During rebase, all other IRC servers apart Darkscience are lost.

Reply:
Yes, this was not an accident :) I was removing them to make life simpler for the testing process.

I see it as an open question how many IRC servers we should have, if and when this gets merged. I don't know but 4 or even 3 seems like too much, but maybe it just depends how the testing goes (I am a bit worried about the mismatch of timing with LN chans being fast and IRC being slow).